### DISCUSSION TITLE
APIドキュメントを自動生成してほしい

### DISCUSSION THREAD

**Discussion by koichimurakami** [mbc-net/mbc-cqrs-serverless#58] [Ideas]
MBC CQRS サーバーレス フレームワークで作成したAPIについてAPIドキュメントを自動生成してほしいです。
コミット後にPipelineでAPIドキュメントを作成してArtifactsにアップロードして頂けますでしょうか？

**Comment by baquan1708** [comment]
@koichimurakami

I've just create PR: https://github.com/mbc-net/mbc-cqrs-serverless/pull/62. When you have a moment, could you please review it?

**Comment by koichimurakami** [comment]
Release Info in Japanese: https://www.mbc-net.com/update-mbc-cqrs-%e3%82%b5%e3%83%bc%e3%83%90%e3%83%bc%e3%83%ac%e3%82%b9-%e3%83%95%e3%83%ac%e3%83%bc%e3%83%a0%e3%83%af%e3%83%bc%e3%82%af-ver-0-1-33-beta-cdk%e3%81%a7%e3%83%87%e3%83%97%e3%83%ad/
